### ISSUE TITLE
Listing saved connections does not work

### ISSUE THREAD

**Issue by atomicsamurai** [rockcarver/frodo-cli#165]
When trying to list existing connections, it does not show the list, but instead prints the following message:

```
$ frodo conn list
Any unique substring of a saved host can be used as the value for host parameter in all commands
```

## Frodo CLI version
```
You seem to be running the NPM package
Installed versions:
cli: v0.19.1
lib: v0.17.1
node: v18.7.0
```

Edit: The command also seems to have deleted all my saved connections.

**Comment by vscheuber** [comment]
hm... weird! I can't dupe:

**Comment by atomicsamurai** [comment]
I am not sure what happened, but I think `frodo conn list` wiped out my `Connections.json`. But its working fine now. Now I am not sure if it was that command or something else!

**Comment by vscheuber** [comment]
@atomicsamurai this is fixed in [v0.18.1-0](https://github.com/rockcarver/frodo-lib/releases/tag/v0.18.1-0), right?
